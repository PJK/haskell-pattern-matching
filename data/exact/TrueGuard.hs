module TrueGuard where

orTrue :: Bool -> Bool
orTrue _ | True = True