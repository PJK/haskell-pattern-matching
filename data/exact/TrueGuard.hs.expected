{
    "tag": "AnalysisSuccess",
    "contents": [
        [
            "orTrue",
            {
                "tag": "NonExhaustive",
                "contents": [
                    [
                        {
                            "tag": "WildcardPattern",
                            "contents": []
                        }
                    ]
                ]
            }
        ]
    ]
}
